com.civicrm.hrjobroles/js/test/filters/isNotUndefined_test.js

```diff
@@ -0,0 +1,28 @@
+define([
+  'common/angular',
+  'job-roles/app'
+], function (angular) {
+  'use strict';
+
+  describe('isNotUndefined', function () {
+    var isNotUndefined;
+
+    beforeEach(module('hrjobroles'));
+    beforeEach(inject(function (isNotUndefinedFilter) {
+      isNotUndefined = isNotUndefinedFilter;
+    }));
+
+    it('should remove undefined values', function () {
+      var array = [1, 2, 'undefined', 'test', undefined];
+
+      expect(isNotUndefined(array).length).toBe(3);
```

Comment:
To be safer, I'd like to check the array contents, like: `expect(isNotUndefined(array)).toEqual(1, 2, 'test');` for example.